src/network/invthread.py

```diff
@@ -75,10 +75,11 @@ def run(self):  # pylint: disable=too-many-branches
                         except KeyError:
                             continue
                         try:
-                            if connection == dandelion.instance.objectChildStem(inv[1]):
+                            childstem, dandelion_enabled = dandelion_ins.objectChildStem(inv[1])
+                            if connection == childstem:
                                 # Fluff trigger by RNG
                                 # auto-ignore if config set to 0, i.e. dandelion is off
-                                if random.randint(1, 100) >= state.dandelion_enabled:  # nosec B311
+                                if random.randint(1, 100) >= dandelion_enabled:  # nosec B311
```

Comment:
we can just use `dandelion_ins.enabled`

Reply:
As per this (https://github.com/Bitmessage/PyBitmessage/pull/2246#discussion_r1618334811) , here `objectChildStem` is returning  value whether dandelion is enabled or not and that is used here. do i need to modify it like it was before?